deb/openmediavault/srv/salt/omv/deploy/avahi/files/etc-avahi-avahi-daemon_conf.j2

```diff
@@ -1,19 +1,22 @@
 {%- set use_ipv4 = salt['pillar.get']('default:OMV_AVAHIDAEMON_USE_IPV4', 'yes') -%}
 {%- set use_ipv6 = salt['pillar.get']('default:OMV_AVAHIDAEMON_USE_IPV6', 'yes') -%}
+{%- set allow_ifaces = salt['pillar.get']('default:OMV_AVAHIDAEMON_ALLOW_IFACES', '') -%}
+{%- set allow_ptp = salt['pillar.get']('default:OMV_AVAHIDAEMON_ALLOW_PTP', 'no') -%}
+{%- set enable_reflector = salt['pillar.get']('default:OMV_AVAHIDAEMON_ENABLE_REFLECTOR', 'no') -%}
 {%- set rlimit_nproc = salt['pillar.get']('default:OMV_AVAHIDAEMON_RLIMIT_NPROC', '3') -%}
 [server]
 #host-name=foo
 #domain-name=local
 #browse-domains=0pointer.de, zeroconf.org
 use-ipv4={{ use_ipv4 }}
 use-ipv6={{ use_ipv6 }}
-#allow-interfaces=eth0
+allow-interfaces={{ allow_ifaces }}
```

Comment:
Wouldn't it make sense to write this only if `allow_ifaces` is not empty. Is there any documentation that clearly tells that it is no problem when `allow-interfaces` is empty?

Reply:
> allow-interfaces= Set a comma seperated list of allowed network interfaces that should be used by the avahi-daemon. Other interfaces will be ignored. If set to the empty list all local interfaces except loopback and point-to-point will be used. 

To me this sounds like `allow-interfaces=` !== `#allow-interfaces=foo`.

Reply:
```suggestion
{%- if allow-interfaces | length > 0 %}
allow-interfaces={{ allow-interfaces }}
{%- endif %}
```

Reply:
That's exactly the reason why I wanted to change that - because documentation is more or less clear in that and it really works like that, so commented out parameter means exactly the empty parameter. You're right that it's not clear enough in their documentation (it sounds like that, but there are still some doubts whether it was really meant like that), but unfortunately, it works like that and to make ptp interfaces work it's needed to have them explicitly added. Thank you for showing me of the filters/conditions, which can be used there - I had no idea about them. It should work in the same way in fact, but it can really avoid any of the misunderstandings of the docs.